rubix-tests/src/test/robotframework/acceptance/bks.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Library     Collections
+Library     OperatingSystem
+Library     Process
+Library     com.qubole.rubix.client.robotframework.BookKeeperClientRFLibrary
+
+*** Keywords ***
+Start BKS
+    [Arguments]         &{options}
+    ${optionArgs} =     Get options argument    &{options}
+    Run                 ${CURDIR}${/}bks.sh start ${optionArgs}
+    Sleep               1s
+
+Stop BKS
+    [Arguments]         &{options}
+    ${optionArgs} =     Get options argument   &{options}
+    Run                 ${CURDIR}${/}bks.sh stop ${optionArgs}
+
+Get options argument
+    [Arguments]         &{options}
+    @{optionsList} =    Create List
+    :FOR    ${key}      IN      @{options.keys()}
+    \   ${value} =      Get From Dictionary     ${options}    ${key}
+    \   Append To List  ${optionsList}  -D${key}\=${value}
+    ${optionArgs} =     Join Command Line   ${optionsList}
+    [Return]            ${optionArgs}
+
+Generate test files
+    [Arguments]     ${numFiles}
+    ...             ${fileLength}
+    @{testFileList} =   Create List
+    :FOR    ${index}    IN RANGE    ${numFiles}
+    \   ${testFile} =        Set variable  ${REMOTE_PATH}${index}
+    \   Generate test file   ${testFile}   ${fileLength}
+    \   Append To List       ${testFileList}    ${testFile}
+    [Return]        @{testFileList}
+
+Read test file data using API
+    [Arguments]     ${fileName}  ${startBlock}  ${endBlock}  ${fileLength}  ${lastModified}  ${clusterType}
+    ${didRead} =    Read data using client api
+    ...             file://${fileName}
+    ...             ${startBlock}
+    ...             ${endBlock}
+    ...             ${fileLength}
+    ...             ${lastModified}
+    ...             ${clusterType}
+    Should be true  ${didRead}
+
+Read test file data using FS
+    [Arguments]     ${fileName}  ${startBlock}  ${endBlock}
+    ${didRead} =    Read data using file system
+    ...             ${fileName}
+    ...             ${startBlock}
+    ...             ${endBlock}
+    Should be true  ${didRead}
+
+Verify cache directories
```

Comment:
Only the cache directory size is being verified. So change the method name to mention that to reduce ambiguity

Reply:
Changed keyword name for clarification